test/jest/rendering.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
We have a similar helper in `test/lib/render-helpers` which we should probably use instead of adding a new file?

Reply:
Hi, I actually deliberately took it from there, because I believe it's better to have all things for Jest in the `test/jest` folder, except the Jest tests themselves. I don't think we should mix shared functions in the same files for Mocha and Jest tests. 

Also, our test folder looks like this:
![image](https://user-images.githubusercontent.com/80175477/115464791-79053f80-a1e2-11eb-93b7-84c8997571e1.png)

There are folders like `data`, `helpers`, `lib`, `mocks`, `stub` and it's not really immediately clear if they have code for Mocha unit tests, Jest unit tests, e2e tests or upcoming Playwright e2e tests or some combination of these. With the `test/jest` folder we can have all shared functions for Jest nicely organized in one place, so it will be easier to do Jest testing.

I will happily search + replace all paths with `test/lib/render-helpers` to `test/jest` for the `renderWithProvider` function and move all Jest-specific shared functions to `test/jest`.